src/app/projects/services/projects.service.ts

```diff
@@ -274,7 +274,9 @@ export class ProjectsService implements OnDestroy {
         ),
         dbgapAccessions: obj.content.dbgap_accessions ?? [],
         publications: obj.publicationsInfo ?? [],
-        authors: obj.publicationsInfo ? obj.publicationsInfo[0].authors : [],
+        authors: obj.publicationsInfo?.length
+          ? obj.publicationsInfo[0].authors || []
+          : [],
```

Comment:
Have you tried this?

```suggestion
        authors: obj.publicationsInfo?.[0]?.authors || []
```

Reply:
That's not valid syntax

Reply:
I didn't realise that was valid syntax, nice